website/docs/flow/index.md

```diff
@@ -20,61 +39,60 @@ Flows can have policies assigned to them. These policies determine if the curren
+-   **Stage configuration**: designates a flow for general setup. This designation doesn't have any constraints in what you can do. For example, by default this designation is used to configure Factors, like change a password and setup TOTP.
 
-#### Recovery
+### Authentication
 
-This designates a flow for recovery. This flow normally contains an [**identification**](stages/identification/) stage to find the user. It can also contain any amount of verification stages, such as [**email**](stages/email/) or [**captcha**](stages/captcha/).
-Afterwards, use the [**prompt**](stages/prompt/) stage to ask the user for a new password and the [**user_write**](stages/user_write.md) stage to update the password.
+Using this option, you can configure whether the the flow requires initial authentication or not, whether the user must be a superuser, or if the flow requires an outpost.
 
-#### Stage configuration
+### Behavior settings
 
-This designates a flow for general setup. This designation doesn't have any constraints in what you can do. For example, by default this designation is used to configure Factors, like change a password and setup TOTP.
+-   **Compatibility mode**: Toggle this option on to increase compatibility with password managers and mobile devices.
 
-## Import & Export
+    Password managers like [1Password](https://1password.com/), for example, don't need this setting to be enabled, when accessing the flow from a desktop browser. However accessing the flow from a mobile device might necessitate this setting to be enabled.
 
-Flows can be imported and exported to share with other people, the community and for troubleshooting. Flows can be imported to apply new functionality and apply existing workflows.
+    The technical reasons for this settings' existence is due to the JavaScript libraries we're using for the default flow interface. These interfaces are implemented using [Lit](https://lit.dev/), which is a modern web development library. It uses a web standard called ["Shadow DOMs"](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM), which makes encapsulating styles simpler. Due to differences in Browser APIs, many password managers are not compatible with this technology.
 
-Download our [Example flows](./examples/flows.md) and then import them into your authentik instance.
+    When the compatibility mode is enabled, authentik uses a polyfill which emulates the Shadow DOM APIs without actually using the feature, and instead a traditional DOM is rendered. This increases support for password managers, especially on mobile devices.
 
-Starting with authentik 2022.8, flows will be exported as YAML, but JSON-based flows can still be imported.
+-   **Denied action**: Configure what happens when access to a flow is denied by a policy. By default, authentik will redirect to a `?next` parameter if set, and otherwise show an error message.
 
-## Behavior settings
```

Comment:
Where?  Where does one "toggle these settings?"

Reply:
Ah, yes, this confirms a concern I developed last week, that it is not at all clear that all of these sections in this page are sections of the UI. On the Create flow modal, there is a section called **Denied action** with these three radio buttons. I am going to look into how to make it more clear on this page that these are UI elements. Any ideas?

Reply:
OK @kensternberg-authentik I reformatted that page to use bold text for each field, instead of H3s... hopefully that makes it more obvious that those are field names, not headers/titles. I also added a screenshot of the Create modal box... to hint that the items below are those fields.